_source/_posts/2019-09-23-tutorial-build-a-maven-plugin.md

```diff
@@ -5,9 +5,9 @@ description: 'Tutorial: How to build a Maven plugin.'
 tags: [tutorial, maven, java, builds]
 author: bdemers
 tweets:
-- "Learn how to build an @ApacheMaven plugin" 
-- "Maven Plugins are built with @ApacheMaven plugins 🐢🐢🐢"
-- "Build a simple @ApacheMaven plugin that will run `git rev-parse`"
+- "Learn how to build an @ASFMavenProject plugin" 
+- "Maven plugins are built with Maven plugins 🐢🐢🐢"
+- "Build a simple ASFMavenProject plugin that will run `git rev-parse`"
```

Comment:
You're missing an @ in front of @ASFMavenProject.

Reply:
doh